ikarus/finiteelements/mechanics/nonlinearelastic.hh

```diff
@@ -138,9 +126,10 @@ namespace Ikarus {
      * @return The strain function calculated using greenLagrangeStrains.
      */
     template <typename ScalarType = double>
-    auto strainFunction(const FERequirementType& par,
-                        const std::optional<const Eigen::VectorX<ScalarType>>& dx = std::nullopt) const {
-      return greenLagrangeStrains(displacementFunction(par, dx));
+    inline auto strainFunction(const FERequirementType& par,
+                               const std::optional<const Eigen::VectorX<ScalarType>>& dx = std::nullopt) const {
+      displacementFunction(par, dx);
```

Comment:
why is this appearing additionally several times?

Reply:
I forgot to remove this at the send. Sorry. Will fix this